integration/tests/bft/gossipsub/rpc_inspector/false_positive_test.go

```diff
@@ -0,0 +1,34 @@
+package rpc_inspector
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+const numOfTestAccounts = 5
+
+type GossipsubRPCInstpectorFalePostiveNotificationsTestSuite struct {
+	Suite
+}
+
+func TestGossipsubRPCInstpectorFalePostiveNotifications(t *testing.T) {
+	suite.Run(t, new(GossipsubRPCInstpectorFalePostiveNotificationsTestSuite))
+}
+
+// TestGossipsubRPCInstpectorFalePostiveNotifications this test ensures that any underlying changes or updates to any of the underlying libp2p libraries
```

Comment:
multiple typos in function name at start of comment (`TestGossipsubRPCInstpectorFalePostiveNotifications` should be `TestGossipsubRPCInspectorFalsePositiveNotifications`) along with actual function name that was already flagged in https://github.com/onflow/flow-go/pull/4371/files#r1210747930

Reply:
https://github.com/onflow/flow-go/pull/4371/commits/05f793b0f611821b359d52c8d59644c01ddbccbb